barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.cpp

```diff
@@ -86,7 +86,10 @@ std::shared_ptr<typename ProverInstances::Instance> ProtoGalaxyProver_<ProverIns
             }
         });
     }
-    next_accumulator->prover_polynomials = std::move(acc_prover_polynomials);
+    for (auto [next_acc_poly, acc_poly] :
```

Comment:
have to move only the polynomials since the rest of the proving key is irrelevant here. This function is definitely need of a refactor (see issue 881 in this function).